### ISSUE TITLE
Voice and SMS reply with "An unknown error occurred"

### ISSUE THREAD

**Issue by Noxcivis** [twilio/OpenVBX#406]
I setup OpenVBX on my hosting provider and went through the setup wizard.  No issues there.  Logged in and imported a Twilio number without problem.  Setup a "flow" for both voice and SMS.  I can use the web interface to call OUT and SMS out but get the robotic "An unknown error occurred" in response to both SMS and voice calls.  Twilio is hitting the website but failing over to the webhook on the "Primary Handler Fails".  

Inbound webhook : https://MYHOSTINGDOMAIN/twilio-OpenVBX-a1d6147/twiml/start/voice/7
The voice fallback is https://MYHOSTINGDOMAIN/twilio-OpenVBX-a1d6147/fallback/voice.php.    

Inbound SMS webhook : https://MYHOSTINGDOMAIN/twilio-OpenVBX-a1d6147/twiml/start/sms/7
The SMS fallback is https://MYHOSTINGDOMAIN/twilio-OpenVBX-a1d6147/fallback/sms.php.

Are there any additional steps I need to take to address this or any recommendations on logs to examine ?  Thanks in advance folks.

**Comment by drosengarden** [comment]
Did you ever figure this out? I have since installed OpenVBX on a different server. I noticed that support for OpenVBX is now deprecated, but I got the software running and sent a test SMS (works)!

However, I cannot seem to translate the webhooks from my old domain url to the new domain url. I figured everything after http://<mydomain/myopenvbxfolder>/twiml/start/sms/3

That's not working - error!

Any advice?
